OutOfSchool/OutOfSchool.IdentityServer/Controllers/AccountController.cs

```diff
@@ -196,6 +196,33 @@ public async Task<IActionResult> SendConfirmEmail()
         return Ok();
     }
 
+    [HttpGet]
+    public async Task<IActionResult> ReSendEmailConfirmation()
+    {
+        var user = await userManager.FindByEmailAsync(User.Identity.Name);
+
+        var token = await userManager.GenerateEmailConfirmationTokenAsync(user);
+        var returnUrl = "Login";
+        var callBackUrl = Url.Action("EmailConfirmation", "Account", new { token, user.Email, returnUrl }, Request.Scheme);
```

Comment:
`"EmailConfirmation", "Account"` - another set of constants used in more than one place. Why not use `nameof(...)` ?